### PR TITLE
feat: add libp2p command, and identify subcommand

### PR DESCRIPTION
**Pull request by aschmahmann**

### CODE REVIEW

File: lib/identify.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+	h, err := libp2pHost()
+	if err != nil {
+		return nil, err
+	}
+
+	if err := h.Connect(ctx, *ai); err != nil {
+		return nil, err
+	}
+
```
Comment (marten-seemann) [inline lib/identify.go:34]: Don't you need to call `IdentifyWait` here?

Comment (aschmahmann) [reply lib/identify.go:34]: Shouldn't that be covered by the BasicHost for me? https://github.com/libp2p/go-libp2p/blob/d3004eaf3cfe2388a39cd0e1bfa76da2153cfe50/p2p/host/basic/basic_host.go#L723-L759

Comment (marten-seemann) [reply lib/identify.go:34]: You're right.

**Comment by aschmahmann** [comment]
Going to merge as is for now since it seems to do the job and is better than nothing 😄. If people have suggestions please file an issue or better yet a PR 🙏

**Pull request merged**